lib/endpoint/tags/table.js

```diff
@@ -130,6 +130,10 @@ function rowToParameter (i, el) {
     params.type = 'integer[]'
   }
 
+  if (params.type === 'array') {
+    params.type = (/_ids$/.test(params.name)) ? 'number[]' : 'string[]'
```

Comment:
ah sorry I missed one thing: we changed `number` to `integer`, so this should be `integer[]`. Can you change that and rebuild the JSON files please? Then it’ll be good to go :)